bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -11,9 +11,8 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
```

Comment:
PR title should be prefixed with the Jira id ([WINC-803](https://issues.redhat.com//browse/WINC-803)).

Also, there is a typo in the commit message: s/rlease/release

Reply:
I suggest using `[bundle]` as the sub-component in the first line of the commit